compiler/env/defines.h

```diff
@@ -82,12 +83,14 @@
 #endif
 
 /*
-  Standarize the OS macros Use HOST_OS instead of various different
+  Standardize the OS macros Use HOST_OS instead of various different
   ways of checking for a particular host os.  As a reference, see
   http://sourceforge.net/p/predef/wiki/OperatingSystems/
 */
 #if defined(__linux__)
 #  define HOST_OS OMR_LINUX
+#elif defined(_TPF_SOURCE)
```

Comment:
Can this be folded as an `|| defined(_TPF_SOURCE)` with `#if defined(__linux__)`?